TestModule.cmake

```diff
@@ -0,0 +1,58 @@
+include (CTest)
+
+set (build_test build_test)
+set (unit_test unit_test)
+set (memcheck memcheck)
+set (complexity_test complexity_test)
+
+if (NOT DEFINED cyclomatic_complexity_warning)
+SET(cyclomatic_complexity_warning 10)
+endif (NOT DEFINED cyclomatic_complexity_warning)
+
+if (test_build)
+enable_testing ()
+
+if (build_test_enabled)
+add_test (
+	NAME ${build_test}
+	COMMAND ${CMAKE_COMMAND} --build .
+)
+endif (build_test_enabled)
+
+if (unit_test_enabled)
+add_executable (${unit_test} ${unit_test_main})
```

Comment:
Outdated project files.

It is advised to copy the template-arduino cmake files from https://github.com/R2D2-2018/Build-environment/tree/master/modules/template-arduino